crowbar_framework/app/models/proposal_queue.rb

```diff
@@ -0,0 +1,40 @@
+class ProposalQueue < ActiveRecord::Base
+  # FIXME: remove this when the export is properly implemented
+  class_attribute :chef_type
+
+  self.chef_type = "data_bag_item"
+  # only accept one default record named "queue"
+  self.primary_key = "name"
+
+  serialize :properties, Utils::JSONWithIndifferentAccess
+
+  def proposals
+    proposal_queue.properties["proposal_queue"]
+  end
+
+  def <<(item)
```

Comment:
Maybe we should replace this with a method named `push`. Then we use the same way as ruby handles arrays, when we need backward we can just set a alias with a depreciation warning.